base/applications/mspaint/winproc.cpp

```diff
@@ -657,6 +689,21 @@ LRESULT CMainWindow::OnCommand(UINT nMsg, WPARAM wParam, LPARAM lParam, BOOL& bH
             if (pd.hDevNames)
                 GlobalFree(pd.hDevNames);
             break;
+        case IDM_FILESEND:
+            if (PathFileExists(g_szFileName))
+            {
+                CStringW strSubject;
+                strSubject.LoadString(IDS_MAILSUBJECT);
+
+                // NOTE: "%22" is for a double quote
+                CString strCmdLine = _T("mailto:?subject=%22");
+                strCmdLine += URLEscapeW(strSubject);
+                strCmdLine += _T("%22&attach=%22");
+                strCmdLine += URLEscapeW(g_szFileName);
+                strCmdLine += _T("%22");
```

Comment:
Alternatively:
```suggestion
                CString strCmdLine;
                strCmdLine.Format(_T("mailto:?subject=%%22%s%%22&attach=%%22%s%%22"),
                                  URLEscapeW(strSubject), URLEscapeW(g_szFileName));
```
Notice that you are mixing TCHARs with explicit unicode strings (from URLEscape**W**) .